R/sankeydiagram.R

```diff
@@ -83,8 +84,8 @@ SankeyDiagram <- function(data = NULL, links.and.nodes = NULL, output.data.only
         # but if table was constructed in other ways, we still want to convert text
         # to factors
         for (i in 1:ncol(data)) {
-            if (is.character(data[[i]]))
-                data[[i]] <- as.factor(data[[i]])
+            if (!is.factor(data[[i]]))
```

Comment:
What case this this change address? Data other than character? Is it for numeric data?

Reply:
Good point. This is for character and QDates. But numeric data should be excluded